K8s/helm/templates/rabbitmq-service-monitor.yaml

```diff
@@ -1,15 +0,0 @@
-apiVersion: monitoring.coreos.com/v1
-kind: ServiceMonitor
-metadata:
-  labels:
-    release: kube-prometheus-stack
-  name: rabbitmq
-spec:
-  endpoints:
-    - path: /metrics
-      targetPort: 15692
```

Comment:
The name of this port is `http-management`. Shouldn't this port name be added to `service-monitor.yaml` so that prom is able to scrape mongodb rabbitmq metrics?

Reply:
Still some work to do on Service Monitor. There's numerous endpoints and ports.

Reply:
Adding `prometheus-metrics` to service monitor port definitions.